specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/mfe.json

```diff
@@ -24624,6 +25045,78 @@
       },
       "additionalProperties": false
     },
+    "ServerlessInferenceEndpoint": {
+      "required": [
+        "uri"
+      ],
+      "type": "object",
+      "properties": {
+        "headers": {
+          "description": "Specifies any required headers to target this serverless endpoint.",
+          "type": "object",
```

Comment:
Please make this read only property here as discussed earlier.

Reply:
readonly is already there, it looks like